packages/client/modules/teamDashboard/components/ProviderRow/GitLabProviderRow.tsx

```diff
@@ -157,34 +157,32 @@ const GitLabProviderRow = (props: Props) => {
       </RowInfo>
       {!auth && (
         <ProviderActions>
-          <>
-            <StyledPrimaryButton
-              onClick={() => openOAuth(cloudProviderId, clientId, serverBaseUrl)}
-              palette='warm'
-              waiting={submitting}
-              onMouseOver={cloudOpenTooltip}
-              onMouseOut={cloudCloseTooltip}
-              ref={cloudRef}
-            >
-              {isDesktop ? 'Connect' : <Icon>add</Icon>}
-            </StyledPrimaryButton>
-            {cloudTooltipPortal('Connect to GitLab Cloud')}
-            {null && (
-              <>
-                <StyledSecondaryButton
-                  onClick={() => console.log('todo')}
-                  palette='warm'
-                  waiting={submitting}
-                  onMouseOver={selfHostedOpenTooltip}
-                  onMouseOut={selfHostedCloseTooltip}
-                  ref={selfHostedRef}
-                >
-                  {isDesktop ? 'Self-hosted' : <Icon>enhanced_encryption</Icon>}
-                </StyledSecondaryButton>
-                {selfHostedTooltipPortal(`Connect to GitLab Self-hosted`)}
-              </>
-            )}
-          </>
+          <StyledPrimaryButton
+            onClick={() => openOAuth(cloudProviderId, clientId, serverBaseUrl)}
+            palette='warm'
+            waiting={submitting}
+            onMouseOver={cloudOpenTooltip}
+            onMouseOut={cloudCloseTooltip}
+            ref={cloudRef}
+          >
+            {isDesktop ? 'Connect' : <Icon>add</Icon>}
+          </StyledPrimaryButton>
+          {/* {cloudTooltipPortal('Connect to GitLab Cloud')} */}
```

Comment:
I think we should hide the tooltip until GitLab Self-hosted is ready. I was wondering why GitLab had a tooltip but all the other integrations didn't

Reply:
+1 I agree, could you add a TODO in the comment?